Makefile

```diff
@@ -303,8 +303,7 @@ db-tools: ## Compile MDBX debugging tools.
 	@echo "Run \"$(DB_TOOLS_DIR)/mdbx_chk\" for the MDBX db file integrity check."
 
 .PHONY: update-book-cli
-update-book-cli: ## Update book cli documentation.
-	cargo build --bin reth --features "$(FEATURES)" --profile "$(PROFILE)"
+update-book-cli: build ## Update book cli documentation.
```

Comment:
this builds in release mode every time by default which is probably not ideal

Reply:
Yeah, and it depends on weather the binary reth changed or not, if nothing changed, I think cargo would not need to recompile the binary

Reply:
should we revert it @onbjerg ?